src/main/java/interfaces/ProduceMeat.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (c) 2022
+ * empty, for future use
+ * use it for free
+ */
+
+package interfaces;
+
+public interface ProduceMeat {
```

Comment:
will you implement this?

Reply:
not yet. It was made in advance. I'm about to delete it.